CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.0.3 - 2019-07-30
```

Comment:
Version should be bumped in `src/mssql.go`

Reply:
oh, thank you! I forgot about it